hotdoc/extensions/devhelp/devhelp_extension.py

```diff
@@ -169,12 +170,20 @@ def __format(self, project):
     def __project_written_out_cb(self, project):
         self.__format(project)
 
+    def __ignore_cb(self, src, files):
+        ignored = set()
+        for f in files:
+            path = Path(src, f).as_posix()
+            if any(path.endswith(i) for i in {'/assets/fonts', '/assets/js/search', '/dumped.trie'}):
```

Comment:
I don't think removing `/assets/fonts` is correct, OK for me otherwise.

Reply:
Though the loop on all files is a bit brutal, have you timed the difference?

Reply:
> Though the loop on all files is a bit brutal, have you timed the difference?

Building doc takes ages with or without that change, that's clearly not the bottleneck.

Reply:
> That's what gst does: https://gitlab.freedesktop.org/gstreamer/gst-docs/-/merge_requests/103. Works fine without fonts here.

OK

> Building doc takes ages with or without that change, that's clearly not the bottleneck.

We don't need to make it take longer then do we? Please just time and report results, I doubt it will be significant but might as well do our due diligence.